source/software/r_package_management.rst

```diff
@@ -12,6 +14,8 @@ very easy for a user to install them just for himself, or for his
 research group, that is not a problem though. Do not hesitate to contact
```

Comment:
Please improve the two sentences above as:

"Some of those are part of the default installation
on VSC infrastructure **as modules**. Given the astounding number of packages, it is
not sustainable to install each and everyone **of them** system wide."

Reply:
I reforumlated this a bit more because this change to me would suggest that the R packages themselves are available as modules. Since the "Some of those" was still refering to the R packages from the previous sentence.